pkg/observable/logging/logging.go

```diff
@@ -16,7 +15,7 @@ func LogErrors(ctx context.Context, errs observable.Observable[error]) {
 
 // forEachErrorLogError is a ForEachFn that logs the given error.
 func forEachErrorLogError(ctx context.Context, err error) {
-	logger := polylog.Ctx(ctx)
-	// Logging the error and flushing (i.e. sending) the log message to stdout
-	logger.Error().Err(err).Send()
+	//logger := polylog.Ctx(ctx)
+	//// Logging the error and flushing (i.e. sending) the log message to stdout
+	//logger.Error().Err(err).Send()
```

Comment:
What's the plan here?

Reply:
![200w](https://github.com/pokt-network/poktroll/assets/1892194/cee88deb-82d8-4917-bc01-3232b3535af3)

Reply:
False go-routine leak alert, will revert this change.